py/gc_long_lived.c

```diff
@@ -85,11 +85,13 @@ mp_obj_property_t *make_property_long_lived(mp_obj_property_t *prop, uint8_t max
     return gc_make_long_lived(prop);
 }
 
+extern const mp_obj_dict_t mcu_module_globals;
+
```

Comment:
this added extern declaration may not be needed, I don't see where it is used.

Reply:
Yup, I added it for debugging iirc. Thanks!